spec/src/main/asciidoc/changelog.asciidoc

```diff
@@ -19,6 +19,14 @@
 
 :numbered!:
 
+=== Release 1.2
+
+* By default excluded MicroProfile Metrics, Health and OpenAPI from tracing (https://github.com/eclipse/microprofile-opentracing/pull/95[#95])
```

Comment:
"By default will not trace endpoints associated with MicroProfile Metrics, Health and OpenAPI (https...."?